### DISCUSSION TITLE
Running a pipline with multiple input files in one go

### DISCUSSION THREAD

**Discussion by hothman** [kedro-org/kedro#2155] [Idea]
Hello, 
Is there a way in Kedro that allows running the same pipeline with multiple input files that will undergo the same treatment? 
Say for example I have: `malaria_dm.csv` and `tuberculosis_dm.csv` and I want to run data cleaning pipline one time instead of two. 

Thanks

**Comment by foxale** [comment]
Why wouldn't you run the pipeline twice? That sounds a bit like [The XY Problem](https://xyproblem.info/).

**Reply by hothman** [reply]
It may sound like an XY problem, but running the pipeline twice is just an example. I actually intended to run the workflow over hundreds of datasets in one go. 
To me, the whole idea of using Kedro is to automate data processing without relying on workflow management systems like Nextflow and Snakemake or even a simple bash script.
